lisa/sut_orchestrator/libvirt/platform.py

```diff
@@ -17,7 +17,7 @@
 from typing import Any, Dict, List, Optional, Tuple, Type, cast
 
 import libvirt  # type: ignore
-import pycdlib
+import pycdlib  # type: ignore
```

Comment:
Any change causes to add it? I haven't figured out when it's needed, just know it from errors.

Reply:
pycdlib doesn't include typing information, so not sure how it wasn't throwing an error before. In this case it's not actually getting installed because it's part of the libvirt extras, but I got the same error either way,.